stuffer/s2n_stuffer_file.c

```diff
@@ -50,6 +52,7 @@ int s2n_stuffer_recv_from_fd(struct s2n_stuffer *stuffer, const int rfd, const u
 
 int s2n_stuffer_send_to_fd(struct s2n_stuffer *stuffer, const int wfd, const uint32_t len, uint32_t *bytes_sent)
 {
+    POSIX_ENSURE_REF(stuffer->blob.data);
```

Comment:
If this precondition was introduced only to avoid erring out on the `write` call, can we instead change line 66 to:

```c
w = (stuffer->blob.data) ? write(wfd, stuffer->blob.data + stuffer->read_cursor, len) : -1;
```

This would not rule out `stuffer->blob.data == NULL` code path entirely, but just make sure that it's correctly handled for `write`.

Reply:
Implemented as suggested.